spec/Cabin/Seat.vspec

```diff
@@ -70,6 +82,25 @@ MassageLevel:
   unit: percent
   description: Seat massage level. 0 = off. 100 = max massage.
 
+MassageStatus:
+  datatype: string
+  type: sensor
+  allowed: ['ON', 'OFF']
+  description: Massage status.  
+
+SupportedMassageTypes:
+  datatype: string[]
+  type: attribute
+  allowed:  ['Off','Roll','Knead','Anti Fatigue','Pulse','Stretch','Lumbar','Shoulder','Wave','Relief','Glide']
+  description:  Type of massage.
+
+MassageType:
+  datatype: string
+  type: actuator
+  allowed:  ['Off','Roll','Knead','Anti Fatigue','Pulse','Stretch','Lumbar','Shoulder','Wave','Relief','Glide']
```

Comment:
Is the idea that if someone turns the massage off they should both:

- Set MassageStatus to OFF
- Set MassageType to OFF

I.e. in practice it should never be so that only one of them are off?